src/main/java/com/mraof/minestuck/blockentity/ComputerBlockEntity.java

```diff
@@ -47,32 +50,42 @@ public ComputerBlockEntity(BlockPos pos, BlockState state)
 	public Hashtable<Integer, String> latestmessage = new Hashtable<Integer, String>();
 	public CompoundTag programData = new CompoundTag();
 	public int programSelected = -1;
+	public List<Block> hieroglyphsStored = new ArrayList<>();
+	public boolean hasParadoxInfoStored = false; //sburb code component received from the lotus flower
+	public int blankDisksStored;
 	
 	@Override
 	public void load(CompoundTag nbt)
 	{
 		super.load(nbt);
-		if (nbt.contains("programs"))
+		if(nbt.contains("programs"))
 		{
 			CompoundTag programs = nbt.getCompound("programs");
-			for (Object name : programs.getAllKeys())
+			for(Object name : programs.getAllKeys())
 			{
-				installedPrograms.put(programs.getInt((String)name), true);
+				installedPrograms.put(programs.getInt((String) name), true);
 			}
 		}
-
+		
 		latestmessage.clear();
-		for(Entry<Integer,Boolean> e : installedPrograms.entrySet())
+		for(Entry<Integer, Boolean> e : installedPrograms.entrySet())
 			if(e.getValue())
 				latestmessage.put(e.getKey(), nbt.getString("text" + e.getKey()));
-
+		
 		programData = nbt.getCompound("programData");
 		
 		if(nbt.contains("ownerId"))
 			ownerId = nbt.getInt("ownerId");
 		else this.owner = IdentifierHandler.load(nbt, "owner");
 		if(gui != null)
 			gui.updateGui();
+		
+		if(nbt.contains("hieroglyphsStored"))
+			hieroglyphsStored = ReadableSburbCodeItem.getRecordedBlocks(nbt.getList("hieroglyphsStored", Tag.TAG_STRING));
+		if(nbt.contains("hasParadoxInfoStored"))
+			hasParadoxInfoStored = nbt.getBoolean("hasParadoxInfoStored");
+		if(nbt.contains("blankDisksStored"))
+			blankDisksStored = nbt.getInt("blankDisksStored");
```

Comment:
Move this above the `updateGui()` call. You want to load hieroglyph and bland disk data before updating the gui. This is the cause of the problem you described in the disk burner class.